android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -213,6 +214,21 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
         if (!questionnaireConfig.type.isDefault()) {
           setQuestionnaireResponse(questionnaireResponse.encodeResourceToString())
         }
+        questionnaireConfig.resourceIdentifier?.takeIf { it.isNotBlank() }?.let {
+          val resourceId = IdType(it)
+          val resourceType =
+            resourceId.resourceType?.let { ResourceType.fromCode(it) }
+              ?: questionnaireConfig.resourceType ?: ResourceType.Patient
```

Comment:
k so this will default to assuming it's a patient, that seems OK, any implications of this we should be concerned about / keep an eye on for the future?